src/handlers/pricing/pricing-label.ts

```diff
@@ -74,10 +74,12 @@ export async function onLabelChangeSetPricing(context: Context): Promise<void> {
     return;
   }
 
+  if (!hasAssistivePricing) return;
```

Comment:
I think that `void` returns are banned in favor of `null`. Are you sure this runs without issue?

Reply:
Yeah, when I tried, there was no issue.